spec/core_functions/color/legacy.hrx

```diff
@@ -0,0 +1,175 @@
+<===> rgb/input.scss
+@use "sass:color";
+a {b: color.is-legacy(rgb(0 255 0))}
+
+<===> rgb/output.css
+a {
+  b: true;
+}
+
+<===>
+================================================================================
+<===> hwb/input.scss
+@use "sass:color";
+a {b: color.is-legacy(hwb(0, 50%, 0%))}
+
+<===> hwb/output.css
+a {
+  b: true;
+}
+
+<===>
+================================================================================
+<===> hsl/input.scss
+@use "sass:color";
+a {b: color.is-legacy(hsl(110, 31%, 32%))}
+
+<===> hsl/output.css
+a {
+  b: true;
+}
+
+<===>
+================================================================================
+<===> srgb/input.scss
```

Comment:
At some point we'll have to decide which of this need to be included to feel like we've covered enough cases. We probably won't need to test all of the color types.